qemu/tests/secure_img.py

```diff
@@ -0,0 +1,75 @@
+from virttest import error_context
+
+
+def run_cmd_in_guest(vm, cmd, test, timeout=60):
+    """
+    Run command in the guest
+
+    :params vm: vm object
+    :params cmd: a command needs to be ran
+    :params session: the vm's session
+    :params status: cmd status
+    :params output: cmd output
+    """
+    session = vm.wait_for_login(timeout=timeout)
+    status, output = session.cmd_status_output(cmd, timeout=timeout)
+    test.log.info("The command of '%s' output: %s", cmd, output)
+    if not status:
+        return output
+    else:
+        test.fail("cmd runs failed cmd: %s, output: %s", cmd, output)
+    session.close()
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1. Create the secure boot params inside of the guest
+    2. Download the certification for secure in side of the guest
```

Comment:
I got confused by the "secure boot params". I thought you were referring to https://www.ibm.com/docs/en/aix/7.2?topic=configuration-secure-boot which is different.

How about the following phrasing?

1. Create the params file with kernel options for the secure image.
2. Download the certificate for the secure image inside of the guest.

Reply:
Updated